packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1117,7 +1117,7 @@ class PopupMenuButtonState<T> extends State<PopupMenuButton<T>> {
         message: widget.tooltip ?? MaterialLocalizations.of(context).showMenuTooltip,
         child: InkWell(
           onTap: widget.enabled ? showButtonMenu : null,
-          canRequestFocus: widget.enabled,
+          canRequestFocus: widget.enabled || NavigationModality.isDirectional(context),
```

Comment:
(side node: I really wish we would have called this property "focusable" instead of "canRequestFocus")

Reply:
Yeah, me to.  It's already in our "I wish we could make a breaking change for this" issue.  I mean, I'm happy to make a breaking change if we think it is important enough, but it would be just a name change.